.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: "Unit Tests"
+
+on:
+  pull_request:
+    branches:
+    - main
+permissions:
+  contents: write
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: Running Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.18.3'
+
+      - name: Setup test
+        run: |
+          go install github.com/onsi/ginkgo/v2/ginkgo
+
```

Comment:
```suggestion
```

Reply:
I don't see any difference here or am I going blind :)

Reply:
Ok.. in email I could see the difference.